data/html5video-rules.json

```diff
@@ -0,0 +1,32 @@
+{
+    "rules": [
+        {
+            "name": "MOV",
+            "displayName" : "MOV",
+            "icon": "",
+            "regex": "<a href="(https|http)?:?(.*\.mov)".*>.*<\/a>",
+            "replacement": "<div class='embed-container' data-src="$1" data-type="video/mp4" data-codec="avc1.42E01E, mp4a.40.2" ><br><video width="100%" height="auto" preload controls><source src="$1" type='video/mp4; codecs="avc1.42E01E, mp4a.40.2"' /></video></div>"
+        },
+        {
+            "name": "MP4",
+            "displayName" : "MP4",
+            "icon": "",
+            "regex": "<a href="(https|http)?:?(.*\.mp4)".*>.*<\/a>",
+            "replacement": "<div class="embed-container"  data-src="$1" data-type="video/mp4" data-codec="avc1.42E01E, mp4a.40.2" ><br><video width="100%" height="auto" preload controls><source src="$1" type='video/mp4; codecs="avc1.42E01E, mp4a.40.2"' /></video></div>"
+        },
+        {
+            "name": "OGV",
+            "displayName" : "OGV",
+            "icon": "",
+            "regex": "<a href="(https|http)?:?(.*\.ogv)".*>.*<\/a>",
```

Comment:
Could you add unit tests?
See: https://github.com/NicolasSiver/nodebb-plugin-ns-embed/blob/0ea86bfd58bdb4fea74b5a3bdfeb35c88eed52c2/test/regexRules.spec.js#L15-L18